README.md

```diff
@@ -49,11 +33,11 @@ trait MySerializable
 Installation:
 
 ```scala
-libraryDependencies += repo %% "akka-serializability-checker-library" % commit
-libraryDependencies += compilerPlugin(repo %% "akka-serializability-checker-plugin" % commit)
+libraryDependencies += "com.github.VirtusLab.akka-safer-serializer" %% "akka-serializability-checker-library" % commit
+libraryDependencies += compilerPlugin("com.github.VirtusLab.akka-safer-serializer" %% "akka-serializability-checker-compiler-plugin" % commit)
 ```
 
-### 3. Dump Schema
+### 2. Dump Event Schema
 
 An sbt plugin that allows for dumping schema of events to a file. Can be used for detecting accidental changes of
```

Comment:
Mention that you mean akka-persistent events, with some relevant link to Akka docs